exercise4_2.py

```diff
@@ -10,17 +10,28 @@
 by Charles R. Severance
 """
 
-repeat_lyrics()
+input_word = input('How much you wanna replay word: ')
 
+try:
+    input = int(input_word)              # Only allows input floats
+except ValueError:
+    print('Error, please enter numeric input')
+    quit()
 
 def repeat_lyrics():
-    print_lyrics()
-    print_lyrics()
-
+    print_lyrics() 
+    #print_lyrics()
+    
 
 def print_lyrics():
-    print("I'm a lumberjack, and I'm okay.")
-    print('I sleep all night and I work all day.')
+    for x in range(input):
+        x = print("I'm a lumberjack, and I'm okay.")
+        x = print('I sleep all night and I work all day.')
```

Comment:
```suggestion
        print("I'm a lumberjack, and I'm okay.")
        print('I sleep all night and I work all day.')
```
`print` returns `None`